doc/source/changed_in_57.rst

```diff
@@ -0,0 +1,257 @@
+.. _changed_in_57:
+
+=============================
+Changed in Percona Server 5.7
+=============================
+
+|Percona Server| 5.7 is based on |MySQL| 5.7 and incorporates many of the improvements found in |Percona Server| 5.6.
+
+Features removed from |Percona Server| 5.7 that were available in |Percona Server| 5.6
+=======================================================================================
+
+Some features that were present in |Percona Server| 5.6 have been removed in |Percona Server| 5.7. These are:
+
+ * `Handlersocket <https://www.percona.com/doc/percona-server/5.6/performance/handlersocket.html>`_ - This feature might be included in a future release if HandlerSocket starts supporting 5.7.
+ * `Support for Fake Changes <https://www.percona.com/doc/percona-server/5.6/management/innodb_fake_changes.html>`_ - Instead of slave prefetching using the fake changes, a 5.7 intra-schema parallel replication slave should be used.
+ * ``SHOW ENGINE INNODB STATUS`` no longer prints the count of active Read-Only transactions.
+ * `InnoDB redo log archiving <https://www.percona.com/doc/percona-server/5.6/management/log_archiving.html>`_ has been removed due to lack of user uptake of the feature.
+
+Changes in Percona Server 5.6 features 
+======================================
+
+ * The minor |Percona Server| version number ("y" in "5.a.b-x.y") has been dropped to simplify |Percona Server| versioning.
+
+ * Performance Schema memory instrumentation support has been added to the :ref:`audit_log_plugin`, :ref:`scalability_metrics_plugin`, and :ref:`pam_plugin`, and to the core server to track memory used by :ref:`user_stats`, :ref:`per_query_variable_statement`, :ref:`changed_page_tracking`, and :ref:`threadpool` features.
+
+ * :ref:`audit_log_plugin` now produces diagnostics in a format consistent with the rest of the server. 
+
```

Comment:
This list has empty lines between items, the other ones do not. Not sure if that affects formatting in any way